DER/DerivativesContracts/FuturesAndForwards.rdf

```diff
@@ -248,7 +258,7 @@
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-der-drc-ff;Forward">
-		<rdfs:subClassOf rdf:resource="&fibo-der-drc-bsc;OverTheCounterInstrument"/>
+		<rdfs:subClassOf rdf:resource="&fibo-der-drc-bsc;OverTheCounterDerivativeInstrument"/>
```

Comment:
Apparently there are forwards which are exchange-traded e.g. LME Forwards and electricity forwards